.github/workflows/test-stellar.yaml

```diff
@@ -0,0 +1,236 @@
+
+      - name: Fund Stellar wallet
+        run: node stellar/faucet --recipient $STELLAR_ADDRESS
+
+      - name: Prepare .env
+        run: |
+          echo "PRIVATE_KEY = '$STELLAR_PRIVATE_KEY'" >> .env
+          echo "ENV = 'local'" >> .env
+          echo "YES = 'true'" >> .env
+
+      ###### Command: Deploy Contract ######
+      - name: Deploy axelar_gateway
+        run: |
+          CONTRACT=axelar_gateway
+          node stellar/deploy-contract deploy $CONTRACT --wasm-path $WASM_DIR/$CONTRACT.optimized.wasm --initialize
+
+      - name: Deploy axelar_operators
+        run: |
+          CONTRACT=axelar_operators
+          node stellar/deploy-contract deploy $CONTRACT --wasm-path $WASM_DIR/$CONTRACT.optimized.wasm --initialize
+
+      ###### Command: Gateway ######
+      # FIXME:
+      # - name: Gateway Approve
+      #   run: |
+      #     node stellar/gateway approve avalanche 0x32034b47cb29d162d9d803cc405356f4ac0ec07fe847ace431385fe8acf3e6e5-2 0x4F4495243837681061C4743b74B3eEdf548D56A5 wallet 0x1234
+
+      - name: Gateway Call Contract
+        run: node stellar/gateway.js call-contract avalanche 0x4F4495243837681061C4743b74B3eEdf548D56A5 0x1234
+
+      - name: Gateway Rotate Signers
+        run: |
+          node stellar/gateway rotate --new-nonce test --signers wallet
+          node stellar/gateway rotate --new-nonce test2 --current-nonce test --signers wallet
+
+      - name: Upgrade gateway contract
+        run: |
+          CONTRACT=axelar_gateway
+          node stellar/deploy-contract.js upgrade $CONTRACT --wasm-path $WASM_DIR/$CONTRACT.optimized.wasm
+
+      ###### Command: Operators ######
+
+      - name: Execute is_operator action
+        run: node stellar/operators.js --action is_operator --args $STELLAR_ADDRESS
+
+      - name: Execute add_operator action
+        run: node stellar/operators.js --action add_operator --args $STELLAR_ADDRESS
+
+      - name: Execute remove_operator action
+        run: node stellar/operators.js --action remove_operator --args $STELLAR_ADDRESS
+
+      # TODO: needs gas-service contract
+      # - name: Execute refund action
+      #   run: node stellar/operators.js --action refund
+
+      # FIXME: find proper way to execute this
+      # - name: Execute execute action
+      #   run: node stellar/operators.js --action execute --target $STELLAR_ADDRESS --method "is_operator"
+
+      # TODO: currently fails
+      # - name: is_operator with estimate_cost
+      #   run: node stellar/operators.js --action is_operator --args $STELLAR_ADDRESS --estimate-cost
+
+      ###### Command: Generate Bindings #####
+      - name: generate operators bindings
+        run: |
+          CONTRACT=axelar_operators
+          node stellar/generate-bindings.js --wasm-path $WASM_DIR/$CONTRACT.optimized.wasm --contract-id  $(jq .chains.stellar.contracts.$CONTRACT.address axelar-chains-config/info/local.json) --output-dir ./stellar/bindings/$CONTRACT
```

Comment:
generate bindings script is not working properly as of now. did you run it sucessfully?